pkg/linksharing/web/static/img/collapse-left.svg

```diff
@@ -0,0 +1,4 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
+	<g><path d="M0,0v512h64V0H0z M384,0v512h128V0H384z M320,128L192,256l128,128V128z"/></g>
+</svg>

```

Comment:
This image has black bars on the side. 

Also the sideways arrow only makes sense on a large viewport.